cdragontoolbox/hashes.py

```diff
@@ -328,6 +328,39 @@ def substitute_basename_words(self):
     def add_basename_word(self):
         super()._add_basename_word(self.build_wordlist())
 
+    def multi_substitution(self, plugin, fileext=""):
+        """Replaces a word in all basenames while adding an additional word near it.
```

Comment:
As said previously on Discord, this method should be made more generic. Current behavior seems very specific.